test/space/space_assertions.go

```diff
@@ -145,6 +152,23 @@ func ProvisioningFailed(msg string) toolchainv1alpha1.Condition {
 	}
 }
 
+func Retargeting() toolchainv1alpha1.Condition {
+	return toolchainv1alpha1.Condition{
+		Type:   toolchainv1alpha1.ConditionReady,
+		Status: corev1.ConditionFalse,
+		Reason: toolchainv1alpha1.SpaceRetargetingReason,
+	}
+}
+
+func RetargetingFailed(msg string) toolchainv1alpha1.Condition {
+	return toolchainv1alpha1.Condition{
+		Type:    toolchainv1alpha1.ConditionReady,
+		Status:  corev1.ConditionFalse,
+		Reason:  toolchainv1alpha1.SpaceRetargetingFailedReason,
+		Message: msg,
+	}
+}
```

Comment:
This function doesn't seem to be used - which means that the failure case is not covered by any of the unit tests.

Reply:
used in c168c90